components/member-profile/index.js

```diff
@@ -118,7 +118,7 @@ const Profile = (props) => {
                 height="16"
                 aria-hidden="true">
                 <path
-                  fill-rule="evenodd"
+                  fillRule="evenodd"
```

Comment:
How do these changes help?

Reply:
Actually I am not sure why this changes came in, strange

Reply:
Always scan your PR once, before sending in for a review :)

Reply:
Noted :)

Reply:
The error is because this issue,
https://stackoverflow.com/questions/39088153/how-do-i-get-rid-of-react-error-warnings-telling-me-to-change-svg-attributes-to

Reply:
`eslint --fix` automatically makes it `fillRule`